API-Lib/src/main/java/de/muenchen/vaadin/demo/apilib/domain/BaseEntity.java

```diff
@@ -15,4 +32,12 @@ public Long getOid() {
     public void setOid(Long oid) {
         this.oid = oid;
     }
+    public String getMandant() {
+        return mandant;
+    }
+
+    public void setMandant(String mandant) {
+        this.mandant = mandant;
+    }
+
 }
```

Comment:
Den hier bitte wieder auf den alten Stand. Der Buerger in der API hat nichts mit dem Buerger aus dem MicroService zu tun